--- android.toolchain.cmake
+++ android.toolchain.cmake
@@ -1322,7 +1322,6 @@ set( ANDROID_RELRO                  ${ANDROID_RELRO}                  CACHE BOOL
 mark_as_advanced( ANDROID_NO_UNDEFINED ANDROID_SO_UNDEFINED ANDROID_FUNCTION_LEVEL_LINKING ANDROID_GOLD_LINKER ANDROID_NOEXECSTACK ANDROID_RELRO )
 
 # linker flags
-set( ANDROID_LINKER_FLAGS "" )
 
 if( ARMEABI_V7A )
  # this is *required* to use the following linker flags that routes around
