--- libs/assimp-3.1.1/CMakeLists.txt	2014-11-13 14:45:09.216454875 -0300
+++ libs/assimp-3.1.1/CMakeLists.txt	2014-11-13 14:45:18.832454438 -0300
@@ -76,7 +76,7 @@
 	"Path the header files are installed to." )
 SET( ASSIMP_BIN_INSTALL_DIR "bin" CACHE PATH
 	"Path the tool executables are installed to." )
-SET ( ASSIMP_BUILD_STATIC_LIB OFF CACHE BOOL
+SET ( ASSIMP_BUILD_STATIC_LIB ON CACHE BOOL
     "Build a static (.a) version of the library" )
 
 SET(ASSIMP_DEBUG_POSTFIX "d" CACHE STRING "Debug Postfitx for lib, samples and tools")
