--- libs/bfl-0.7.0/CMakeLists.txt	2014-12-23 14:57:42.998174978 -0300
+++ libs/bfl-0.7.0/CMakeLists.txt	2014-12-23 14:58:22.034173201 -0300
@@ -54,7 +54,7 @@
 # Shared or static library
 # ------------------------
 IF (NOT LIBRARY_TYPE)
-  SET( LIBRARY_TYPE shared CACHE STRING "What type of library to build: (shared, static or both)")
+  SET( LIBRARY_TYPE static CACHE STRING "What type of library to build: (shared, static or both)")
 ENDIF (NOT LIBRARY_TYPE)
 
 
