--- catkin_ws/src/image_common/camera_info_manager/CMakeLists.txt	2014-12-12 12:17:14.245252860 -0300
+++ catkin_ws/src/image_common/camera_info_manager/CMakeLists.txt	2014-12-12 12:17:26.965252568 -0300
@@ -33,7 +33,7 @@
         DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
 )
 
-if(CATKIN_ENABLE_TESTING)
+if(CATKIN_ENABLE_TESTING AND NOT ANDROID)
   find_package(rostest)
   # Unit test uses gtest, but needs rostest to create a ROS environment.
   # Hence, it must be created as a normal executable, not using
