--- catkin_ws/src/image_common/camera_calibration_parsers/CMakeLists.txt	2015-06-11 00:15:52.835902346 -0400
+++ catkin_ws/src/image_common/camera_calibration_parsers/CMakeLists.txt	2015-06-11 00:22:10.574287606 -0400
@@ -18,6 +18,9 @@
   find_library(YAML_CPP_LIBRARIES NAMES yaml-cpp)
   find_path(YAML_CPP_H_INCLUDE_DIR yaml-cpp/yaml.h )
   SET(YAML_CPP_INCLUDE_DIR ${YAML_CPP_H_INCLUDE_DIR}/yaml-cpp)
+elseif (ANDROID)
+  find_package(yaml-cpp)
+  add_definitions(-DHAVE_NEW_YAMLCPP)
 else()
   pkg_check_modules(YAML_CPP yaml-cpp)
   if(${YAML_CPP_VERSION} VERSION_GREATER 0.5)
