--- libs/collada-dom-2.4.0/dom/CMakeLists.txt	2012-06-01 08:43:20.000000000 -0400
+++ libs/collada-dom-2.4.0/dom/CMakeLists.txt	2014-11-11 10:50:50.189100025 -0300
@@ -27,7 +27,7 @@
   install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/1.4 DESTINATION ${COLLADA_DOM_INCLUDE_INSTALL_DIR} COMPONENT ${COMPONENT_PREFIX}-dev  PATTERN ".svn" EXCLUDE PATTERN ".~" EXCLUDE)
 endif()
 
-add_library(collada-dom SHARED ${COLLADA_BASE_SOURCES})
+add_library(collada-dom ${COLLADA_BASE_SOURCES})
 target_link_libraries(collada-dom ${COLLADA_LIBS})
 set_target_properties(collada-dom PROPERTIES
   COMPILE_FLAGS "${COLLADA_COMPILE_FLAGS}"
