--- catkin_ws/src/robot_model/collada_parser/CMakeLists.txt	2014-11-12 16:50:45.229284101 -0300
+++ catkin_ws/src/robot_model/collada_parser/CMakeLists.txt	2014-11-12 16:56:55.977275596 -0300
@@ -26,11 +26,11 @@
 
 # necessary for collada reader to create the temporary dae files due
 # to limitations in the URDF geometry
-include (CheckFunctionExists)
-check_function_exists(mkstemps HAVE_MKSTEMPS)
-if(HAVE_MKSTEMPS)
-  add_definitions("-DHAVE_MKSTEMPS")
-endif()
+#include (CheckFunctionExists)
+#check_function_exists(mkstemps HAVE_MKSTEMPS)
+#if(HAVE_MKSTEMPS)
+#  add_definitions("-DHAVE_MKSTEMPS")
+#endif()
 
 # build the parser lib
 add_library(${PROJECT_NAME} src/collada_parser.cpp)
