--- catkin_ws/src/vision_opencv/cv_bridge/CMakeLists.txt	2014-12-16 13:23:41.106085276 -0300
+++ catkin_ws/src/vision_opencv/cv_bridge/CMakeLists.txt	2014-12-16 13:25:29.358088880 -0300
@@ -3,7 +3,11 @@
 
 find_package(catkin REQUIRED COMPONENTS rosconsole sensor_msgs)
 
+if(NOT ANDROID)
 find_package(Boost REQUIRED python)
+else()
+find_package(Boost REQUIRED)
+endif()
 find_package(OpenCV REQUIRED)
 if (OpenCV_VERSION VERSION_EQUAL "3")
   add_definitions("-DOPENCV3=1")
@@ -21,7 +25,9 @@
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS} ${OpenCV_INCLUDE_DIRS})
 include_directories(include ${catkin_INCLUDE_DIRS})
 
+if(NOT ANDROID)
 add_subdirectory(python)
+endif()
 add_subdirectory(src)
 if(CATKIN_ENABLE_TESTING)
   add_subdirectory(test)
--- catkin_ws/src/vision_opencv/cv_bridge/src/CMakeLists.txt	2014-12-12 14:16:26.261088783 -0300
+++ catkin_ws/src/vision_opencv/cv_bridge/src/CMakeLists.txt	2014-12-12 14:17:17.117087617 -0300
@@ -5,6 +5,7 @@
 
 install(TARGETS ${PROJECT_NAME} DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
 
+if(NOT ANDROID)
 # add a Boost Python library
 find_package(PythonInterp REQUIRED)
 find_package(PythonLibs "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
@@ -54,3 +55,4 @@
 endif()
 
 install(TARGETS ${PROJECT_NAME}_boost DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}/boost/)
+endif()
