--- libs/fcl-0.3.2/CMakeLists.txt	2014-10-28 02:12:50.000000000 -0300
+++ libs/fcl-0.3.2/CMakeLists.txt	2015-01-15 11:59:07.636226183 -0300
@@ -20,7 +20,7 @@
 if(MSVC OR IS_ICPC)
   option(FCL_STATIC_LIBRARY "Whether the FCL library should be static rather than shared" ON)
 else()
-  option(FCL_STATIC_LIBRARY "Whether the FCL library should be static rather than shared" OFF)
+  option(FCL_STATIC_LIBRARY "Whether the FCL library should be static rather than shared" ON)
 endif()
 
 # Whether to enable SSE
@@ -39,6 +39,8 @@
 set(FCL_HAVE_OCTOMAP 0)
 if(PKG_CONFIG_FOUND)
     pkg_check_modules(OCTOMAP QUIET octomap)
+else()
+    find_package(octomap QUIET)
 endif()
 if(NOT OCTOMAP_FOUND)
     # if pkfconfig is not installed, then fall back on more fragile detection
@@ -82,7 +84,7 @@
 # endif()
 
 
-find_package(Boost COMPONENTS thread date_time filesystem system unit_test_framework REQUIRED)
+find_package(Boost COMPONENTS thread date_time filesystem system REQUIRED)
 include_directories(${Boost_INCLUDE_DIR})
 
 if(MSVC)
@@ -93,6 +95,10 @@
 # FCL's own include dir should be at the front of the include path
 include_directories(BEFORE "include")
 
+set(CCD_FOUND "1")
+set(CCD_INCLUDE_DIRS "/opt/roscpp_output/target/include")
+set(CCD_LIBRARIES "/opt/roscpp_output/target/lib/libccd.a")
+set(CCD_LIBRARY_DIRS "/opt/roscpp_output/target/lib")
 if(PKG_CONFIG_FOUND)
   pkg_check_modules(CCD ccd)
   # check to see if the pkg is installed under the libccd name
@@ -128,11 +134,11 @@
   PATTERN ".DS_Store" EXCLUDE
 )
 
-find_package(catkin)
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME}
-  )
+#find_package(catkin)
+#catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES ${PROJECT_NAME}
+#  )
 
 #install(FILES "${pkg_conf_file}" DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
 
