--- catkin_ws/src/image_pipeline/image_view/CMakeLists.txt	2015-04-06 09:37:51.000000000 -0400
+++ catkin_ws/src/image_pipeline/image_view/CMakeLists.txt	2015-06-11 01:09:23.476120010 -0400
@@ -17,7 +17,9 @@
                            ${GTK2_INCLUDE_DIRS}
                            ${OpenCV_INCLUDE_DIRS}
 )
-add_definitions(-DHAVE_GTK)
+if (NOT ANDROID)
+  add_definitions(-DHAVE_GTK)
+endif()
 
 
 # Nodelet library
