--- catkin_ws/src/moveit_core/CMakeLists.txt	2015-01-14 14:27:44.525016174 -0300
+++ catkin_ws/src/moveit_core/CMakeLists.txt	2015-01-14 13:51:35.113057099 -0300
@@ -28,9 +28,12 @@
 find_package(urdfdom REQUIRED)
 find_package(urdfdom_headers REQUIRED)
 
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(LIBFCL REQUIRED fcl)
-find_library(LIBFCL_LIBRARIES_FULL ${LIBFCL_LIBRARIES} ${LIBFCL_LIBRARY_DIRS})
+#find_package(PkgConfig REQUIRED)
+#pkg_check_modules(LIBFCL REQUIRED fcl)
+set(LIBFCL_LIBRARIES "/opt/roscpp_output/target/lib/libfcl.a")
+set(LIBFCL_LIBRARY_DIRS "/opt/roscpp_output/target/lib")
+set(LIBFCL_LIBRARIES_FULL "/opt/roscpp_output/target/lib/libfcl.a")
+#find_library(LIBFCL_LIBRARIES_FULL ${LIBFCL_LIBRARIES} ${LIBFCL_LIBRARY_DIRS})
 set(LIBFCL_LIBRARIES "${LIBFCL_LIBRARIES_FULL}")
 
 set(VERSION_FILE_PATH "${CATKIN_DEVEL_PREFIX}/include")
