--- libs/orocos_kdl-1.3.0/src/CMakeLists.txt	2015-01-08 20:51:07.591185309 -0300
+++ libs/orocos_kdl-1.3.0/src/CMakeLists.txt	2015-01-08 20:55:49.219178848 -0300
@@ -52,12 +52,12 @@
 #In Windows (Visual Studio) it is necessary to specify the postfix
 #of the debug library name and no symbols are exported by kdl, 
 #so it is necessary to compile it as a static library
-IF(MSVC)
+IF(MSVC OR ANDROID)
     SET(CMAKE_DEBUG_POSTFIX "d")
     SET(LIB_TYPE STATIC)
-ELSE(MSVC)
+ELSE(MSVC OR ANDROID)
     SET(LIB_TYPE SHARED)
-ENDIF(MSVC)
+ENDIF(MSVC OR ANDROID)
 
 CONFIGURE_FILE(config.h.in config.h @ONLY)
 
--- libs/orocos_kdl-1.3.0/models/CMakeLists.txt	2015-01-08 20:54:20.287180889 -0300
+++ libs/orocos_kdl-1.3.0/models/CMakeLists.txt	2015-01-08 20:52:59.587182740 -0300
@@ -2,7 +2,7 @@
 
 IF(BUILD_MODELS)
   
-  ADD_LIBRARY(orocos-kdl-models SHARED puma560.cpp kukaLWR_DHnew.cpp) 
+  ADD_LIBRARY(orocos-kdl-models puma560.cpp kukaLWR_DHnew.cpp) 
   INCLUDE_DIRECTORIES(${PROJ_SOURCE_DIR}/src ${PROJ_BINARY_DIR}/src)
   SET_TARGET_PROPERTIES( orocos-kdl-models PROPERTIES
     SOVERSION "${KDL_VERSION_MAJOR}.${KDL_VERSION_MINOR}"
--- libs/orocos_kdl-1.3.0/src/chainiksolverpos_lma.hpp	2015-01-08 19:26:27.511301853 -0300
+++ libs/orocos_kdl-1.3.0/src/chainiksolverpos_lma.hpp	2015-01-08 19:27:06.679300955 -0300
@@ -91,7 +91,7 @@
      */
     ChainIkSolverPos_LMA(
     		const KDL::Chain& _chain,
-    		const Eigen::Matrix<double,6,1>& _L,
+    		const Eigen::Matrix<double,6,1>& _G,
     		double _eps=1E-5,
     		int _maxiter=500,
     		double _eps_joints=1E-15
--- libs/orocos_kdl-1.3.0/src/chainiksolverpos_lma.cpp	2015-01-08 19:27:36.643300267 -0300
+++ libs/orocos_kdl-1.3.0/src/chainiksolverpos_lma.cpp	2015-01-08 20:26:46.211218835 -0300
@@ -49,7 +49,7 @@
 
 ChainIkSolverPos_LMA::ChainIkSolverPos_LMA(
 		const KDL::Chain& _chain,
-		const Eigen::Matrix<double,6,1>& _L,
+		const Eigen::Matrix<double,6,1>& _G,
 		double _eps,
 		int _maxiter,
 		double _eps_joints
@@ -62,7 +62,7 @@
 	maxiter(_maxiter),
 	eps(_eps),
 	eps_joints(_eps_joints),
-	L(_L.cast<ScalarType>()),
+	L(_G.cast<ScalarType>()),
 	chain(_chain),
 	T_base_jointroot(_chain.getNrOfJoints()),
 	T_base_jointtip(_chain.getNrOfJoints()),
