--- catkin_ws/src/robot_state_publisher/CMakeLists.txt	2016-05-10 16:10:02.284800164 -0400
+++ catkin_ws/src/robot_state_publisher/CMakeLists.txt	2016-05-10 16:58:36.717713582 -0400
@@ -59,6 +63,7 @@
 #rosbuild_download_test_data(http://pr.willowgarage.com/data/robot_state_publisher/joint_states_indexed.bag test/joint_states_indexed.bag 793e0b566ebe4698265a936b92fa2bba)
 
 install(TARGETS ${PROJECT_NAME}_solver ${PROJECT_NAME} state_publisher
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 
