--- catkin_ws/src/navigation/robot_pose_ekf/CMakeLists.txt	2016-05-10 16:10:02.440800597 -0400
+++ catkin_ws/src/navigation/robot_pose_ekf/CMakeLists.txt	2016-05-10 16:43:05.032555201 -0400
@@ -2,8 +2,15 @@
 project(robot_pose_ekf)
 
 # bfl (Bayesian Filtering Library) is a third party package that uses pkg-config
+if(NOT ANDROID)
 find_package(PkgConfig)
 pkg_check_modules(BFL REQUIRED orocos-bfl)
+else()
+set(BFL_INCLUDE_DIRS "/opt/roscpp_output/target/include/bfl")
+set(BFL_LIBRARY_DIRS "/opt/roscpp_output/target/lib")
+set(BFL_LIBRARY_DIRS "/opt/roscpp_output/target/lib")
+set(BFL_LIBRARIES "/opt/roscpp_output/target/lib/liborocos-bfl.a")
+endif()
 
 include_directories(${BFL_INCLUDE_DIRS})
 link_directories(${BFL_LIBRARY_DIRS})
@@ -69,7 +76,7 @@
 ## Tests are failing on OSX for an unknown reason
 include(CMakeDetermineSystem)
 if(CMAKE_SYSTEM_NAME MATCHES "Linux")
-if(CATKIN_ENABLE_TESTING)
+if(CATKIN_ENABLE_TESTING AND NOT ANDROID)
 
 catkin_download_test_data(
   download_data_ekf_test2_indexed.bag
@@ -104,5 +111,5 @@
 add_rostest(${CMAKE_CURRENT_SOURCE_DIR}/test/test_robot_pose_ekf.launch)
 add_rostest(${CMAKE_CURRENT_SOURCE_DIR}/test/test_robot_pose_ekf_zero_covariance.launch)
 
-endif(CATKIN_ENABLE_TESTING)
+endif(CATKIN_ENABLE_TESTING AND NOT ANDROID)
 endif(CMAKE_SYSTEM_NAME MATCHES "Linux")
