--- catkin_ws/src/ros/roslib/src/package.cpp	2014-10-20 23:09:44.306052059 -0300
+++ catkin_ws/src/ros/roslib/src/package.cpp	2014-10-17 10:03:35.026897481 -0300
@@ -48,7 +48,7 @@
 {
   boost::mutex::scoped_lock lock(librospack_mutex);
 
-  rospack::ROSPack rp;
+  /*rospack::ROSPack rp;
   int ret;
   try
   {
@@ -64,7 +64,7 @@
   {
     if ( !rp.is_quiet() )
       std::cerr << "[rospack] " << e.what() << std::endl;
-  }
+  }*/
   return std::string("");
 }
 
