--- catkin_ws/src/robot_model/urdf/CMakeLists.txt	2015-04-22 21:13:36.000000000 -0400
+++ catkin_ws/src/robot_model/urdf/CMakeLists.txt	2015-06-11 15:05:53.450725920 -0400
@@ -10,8 +10,12 @@
 
 find_package(TinyXML REQUIRED)
 
-find_package(PkgConfig)
-pkg_check_modules(libpcrecpp libpcrecpp)
+if (ANDROID)
+  find_package(libpcrecpp)
+else()
+  find_package(PkgConfig)
+  pkg_check_modules(libpcrecpp libpcrecpp)
+endif()
 
 catkin_package(
   LIBRARIES ${PROJECT_NAME}
